Jenkinsfile

```diff
@@ -1,6 +1,9 @@
 buildMvn {
   publishModDescriptor = 'yes'
-  publishAPI = 'yes'
+  doApiDoc = true
+  doApiLint = true
+  apiTypes = 'RAML'
+  apiDirectories = 'ramls'
```

Comment:
No need of `publishAPI`? Maybe devops folks can comment on this change. @funkymalc, @wafschneider, @dcrossleyau, and @ihardy ?

Reply:
That is correct. Explained in the documentation, Start from https://dev.folio.org/reference/api/#explain-api-doc

Reply:
However @julianladisch the "runLintRamlCop" in Jenkinsfile is deprecated by doApiLint and so also needs be removed: https://dev.folio.org/reference/api/#explain-api-lint

Reply:
Thanks, I've removed runLintRamlCop.

Reply:
Thanks for the pointer @dcrossleyau. It would be nice the Jenkinfiles for all modules can be updated to be consistent. Is this typically handled by module developers or devops? BTW I noticed this one use `mvnDeploy = 'yes'` and another module (mod-notes) uses `mvnDeploy = true`.

Reply:
The developers should do the upgrades, i reckon. Our small FOLIO DevOps team does not have the resources.

Also, developers know the module better. As explained at https://dev.folio.org/guides/api-lint/#interpretation-of-messages they could encounter some new violations from this better tool.

Regarding the Jenkinsfile syntax, see https://dev.folio.org/guides/jenkinsfile/
... true or false (or the old syntax 'yes' or 'no')